Information/CenterOfMass.FCMacro

```diff
@@ -114,6 +117,9 @@ class MaterialWidget():
     def on_densitySpinBox_valueChanged(self, density):
         self.combo.setCurrentIndex(0)
         self.parent.on_pushButton_ComputeCDG()
+        self.solid.ViewObject.ShapeColor=(0.8,0.8,0.8)
```

Comment:
Does this have a permanent effect on the shapes? If yes, I think it would be better to save the original color and revert to it upon exiting.

Reply:
It will be reset to default color if any change of density is done.
But you're right, I'll be better to save the original color.
I'll change it